client/src/components/ManageCustomer.js

```diff
@@ -0,0 +1,62 @@
+import "./AutoSuggest.css";
```

Comment:
What is this imported for?

Reply:
I dont think I have imported a css file like this. I doubt that it was automotically added by some plugins I have tried last week. Sorry about that, I think I can delete them as I just checked the "AutoSuggest.css" file on my local editor, there is nothing major style changes regarding the two new js files.

Reply:
Yes delete them if you don't need them